src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -227,6 +227,22 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         }
     };
 
+    handleAnnotationDelete = ({ id }: { id: string, permissions?: BoxAnnotationPermission }) => {
+        const { api, emitRemoveEvent, file } = this.props;
+
+        api.getFeedAPI(false).deleteAnnotation(
+            file,
+            id,
+            () => {
```

Comment:
This might make it a little harder to test with mock functions -- would it be better to create a named function?